packages/pluggableWidgets/floating-action-button-native/src/FloatingActionButton.tsx

```diff
@@ -81,7 +81,7 @@ export class FloatingActionButton extends Component<FloatingActionButtonProps<Fl
                         size={this.styles.secondaryButton.size}
                         title={button.caption && button.caption.value}
                         shadowStyle={this.styles.secondaryButton}
-                        buttonColor={this.styles.secondaryButton.backgroundColor}
+                        buttonColor={this.styles.secondaryButton.backgroundColor as string}
```

Comment:
Howcome this assertion is necessary?

Reply:
new types for RN says the property can be unknown

Reply:
Unknown or `undefined`? If it is `string | undefined` and we know it's definitely a `string`, is `!` not better here?

Reply:
![image](https://user-images.githubusercontent.com/45102481/98347113-4510d600-2017-11eb-8550-64c2bd6a304b.png)

! doesnt resolve the problem, they introduce a new typeof OpaqueColorValue

Reply:
as string was the non painful option